### PR TITLE
bind to 127 with the exception of Kong proxy ports

### LINKED ISSUE #190: Exposed ports of microservices need to be disabled in docker compose file
Currently microservices in EdgeX expose ports ( e.g, 48080 etc) without protection. With proxy in place for Fuji release, these ports need to be disabled and all the access to microservices need to go through proxy.

### PR DESCRIPTION
**Pull request by hutchic**
with the exception of the Kong proxy ports bind all services to 127

I believe this would close #190

### CODE REVIEW

File: releases/fuji/compose-files/docker-compose-fuji-no-secty.yml [hunk 1]
```diff
@@ -50,8 +50,8 @@ services:
   consul:
     image: consul:1.3.1
     ports:
-      - "8400:8400"
-      - "8500:8500"
+      - "127.0.0.1:8400:8400"
```
Comment (lenny-intel) [inline releases/fuji/compose-files/docker-compose-fuji-no-secty.yml:53]: I don't think the Fuji compose files are in scope for this issue.

**Comment by lenny-intel** [comment]
> Not sure we what the Fuji compose files in scope for these changes.

If we do, I think they should be in a separate PR. I am good with this PR w/o the Fuji chnages.

**Comment by bnevis-i** [comment]
Ditto.

**Comment by lenny-intel** [comment]
@hutchic , Please remove the changes to the Fuji compose files from this PR. We can discuss that as a separate PR if community agrees it needs to be retrofitted. THX!

**Pull request merged**